### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [altairwei/WizNotePlus#239] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by altairwei** [comment, marked as answer]
Yes, this project is active. It is great to see that you are interested in this project.

**Reply by ghost** [reply]
Hi altairwei.

> Yes, this project is active. It is great to see that you are interested in this project.

thank you for feedback. I will close this issue, you clarified my doubt.

*Have a great day, and thank you for your patience in answering my question.*
